src/Core/Analyser/EventHandler/DependsOnInternalToken.php

```diff
@@ -29,11 +39,17 @@ public function invoke(ProcessEvent $event): void
         foreach ($event->dependentLayers as $dependentLayer => $_) {
             if ($event->dependerLayer !== $dependentLayer
                 && $event->dependentReference instanceof ClassLikeReference
-                && ($event->dependentReference->hasTag('@deptrac-internal')
-                    || $event->dependentReference->hasTag('@internal'))
             ) {
-                $this->eventHelper->addSkippableViolation($event, $ruleset, $dependentLayer, $this);
-                $event->stopPropagation();
+                $isInternal = $event->dependentReference->hasTag('@deptrac-internal');
+
+                if (!$isInternal && $this->internalTag) {
```

Comment:
`!$isInternal && $this->internalTag !== null`

Reply:
Just in case the internal tag is "0"? Ok ;)